specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/AppResiliency.json

```diff
@@ -0,0 +1,190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerApps API Client",
+    "version": "2023-08-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{appName}/resiliencyPolicies/{name}": {
+      "put": {
+        "tags": [
+          "AppResiliency"
+        ],
+        "summary": "Create or update an application's resiliency policy.",
+        "description": "Create or update an application's resiliency policy.",
+        "operationId": "AppResiliency_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "appName",
+            "description": "Name of the Container App.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "name",
+            "description": "Name of the resiliency policy.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resiliencyEnvelope",
+            "in": "body",
+            "description": "The resiliency policy to create or update.",
+            "required": true,
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/AppResiliency"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
PUT must respond with 200 and 201 status codes

Reply:
Done.